website/docs/docs/cloud/connect-data-platform/connect-snowflake.md

```diff
@@ -30,31 +30,36 @@ to authenticate dbt Cloud to run queries against Snowflake on behalf of a Snowfl
 <Lightbox src="/img/docs/dbt-cloud/snowflake-userpass-auth.png" title="Snowflake username/password authentication"/>
 
 ### Key Pair
+
 **Available in:** Development environments,  Deployment environments
 
 The `Keypair` auth method uses Snowflake's [Key Pair Authentication](https://docs.snowflake.com/en/user-guide/python-connector-example.html#using-key-pair-authentication) to authenticate Development or Deployment credentials for a dbt Cloud project.
 
-After [generating an encrypted key pair](https://docs.snowflake.com/en/user-guide/key-pair-auth.html#configuring-key-pair-authentication), be sure to set the `rsa_public_key` for the Snowflake user to authenticate in dbt Cloud:
+1. After [generating an encrypted key pair](https://docs.snowflake.com/en/user-guide/key-pair-auth.html#configuring-key-pair-authentication), be sure to set the `rsa_public_key` for the Snowflake user to authenticate in dbt Cloud:
 
 ```sql
 alter user jsmith set rsa_public_key='MIIBIjANBgkqh...';
 ```
 
-Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
-Credentials" page to finish configuring dbt Cloud to authenticate with Snowflake
-using a key pair.
-
-**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud, and the keys must be of size 4096 or smaller.
+2. Finally, set the **Private Key** and **Private Key Passphrase** fields in the **Edit Credentials** page to finish configuring dbt Cloud to authenticate with Snowflake using a key pair.
```

Comment:
What is the **Edit Credentials** page?

Reply:
this is live in the docs but i think the person who wrote this means the 'development credentials' page 
![Screenshot 2023-09-25 at 11 25 00](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/9e5f09a6-30b0-42e7-bbb4-bd202e93ad6d)

Reply:
In the vast majority of cases though Key/Pair is used for deployment environment credentials though (rather than Dev credentials), so I would focus the wording on deployment environments.